### PR TITLE
feat: initial style-dictionary implementation for design tokens

### PR DESCRIPTION
**Pull request by dannyk3941**
#### What's this PR do?
- Added `style-dictionary` package and `config.js` file added. Previous color tokens replaced by ones generated from `ds-css-v2/styles/tokens`.
- Folder structure changed in `ds-css-v2/styles/sass` with the addition of `/base` folder used for global styling and `/utilities` for any helper `scss`. 
- All CBP `scss` imports moved from `ds-css-v2/styles/sass/_uswds-theme-custom-styles.scss` to `ds-css-v2/styles/sass/styles.scss` 
#### Where should the reviewer start?
- `ds-css-v2/config.js`
- `ds-css-v2/styles/tokens`
- `ds-css-v2/styles/sass/cbp-tokens`
- `ds-css-v2/styles/sass/styles.scss` 
#### How should this be manually tested?
- Pull branch and run `$ npm install`
- Make sure the command `$ npm run style-dictionary` runs successfully
- Check that the tokens work in scss files. 
#### Any background context you want to provide?
- The purpose of this PR is to centralize color tokens in the project so that it can be handled by `style-dictionary` and use of `JSON` files.
#### What are the relevant tickets?
- JIRA
#### Screenshots (if appropriate)
- n/a
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?

### CODE REVIEW

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss [hunk 1]
```diff
@@ -357,7 +357,7 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
       height: 100vh;
       left: 0;
       max-width: 100vw;
-      padding: $cbp-size-4;
+      padding: 32px; // 0 $cbp-size-4;
```
Comment (ray760) [inline ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:360]: @dannyk3941 Is there no pixel token for 32px?

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss [hunk 2]
```diff
@@ -244,7 +244,7 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
 
   ul {
     padding: 0;
-    margin-top: $cbp-size-1;
+    margin-top: 8px; // $cbp-size-1;
```
Comment (ray760) [inline ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:247]: @dannyk3941 Is there no pixel token for 8px?

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss [hunk 3]
```diff
@@ -183,17 +183,17 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
...
   }
 
   //Recently Visited Applications text
   .cbp-header-subtext {
     display: inline-block;
-    color: $cbp-accent-primary-light;
+    color: $cbp-blue-cool-30; // $cbp-accent-primary-light;
     font-style: italic;
-    margin: $cbp-size-2 $cbp-size-2 0 $cbp-size-2;
+    margin: 16px 16px 0 16px; // $cbp-size-2 $cbp-size-2 0 $cbp-size-2;
```
Comment (ray760) [inline ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:196]: @dannyk3941 Is there no pixel token for 16px?

**Review by ray760 — APPROVED** [review]
Left a view comments about sizing, but other than that It's approved.

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss (hunk 1 above)
Comment (dannyk3941) [reply ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:360]: Resolved. The sizing tokens will be replaced once they are determined by the team.

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss (hunk 2 above)
Comment (dannyk3941) [reply ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:247]: Resolved. The sizing tokens will be replaced once they are determined by the team.

File: ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss (hunk 3 above)
Comment (dannyk3941) [reply ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss:196]: Resolved. The sizing tokens will be replaced once they are determined by the team.

**Pull request merged**